pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [build-system]
 requires = [
   # sync with setup.py until we discard non-pep-517/518
-  "setuptools>=42.0",
-  "setuptools-scm[toml]>=3.4",
+  "setuptools>=50.3.2",
```

Comment:
Can you explain why bumping these is needed?

Reply:
yeah, I went into courage updating the versions. I agree, customer's facing requirements should not be touched without  the need. Will revert it.